tools/ci/od_lints.dm

```diff
@@ -0,0 +1,32 @@
+//1000-1999
+#pragma FileAlreadyIncluded error
+#pragma MissingIncludedFile error
+#pragma MisplacedDirective error
+#pragma UndefineMissingDirective error
+#pragma DefinedMissingParen error
+#pragma ErrorDirective error
+#pragma WarningDirective error
+#pragma MiscapitalizedDirective error
+
+//2000-2999
+#pragma SoftReservedKeyword error
+#pragma DuplicateVariable error
+#pragma DuplicateProcDefinition error
+#pragma TooManyArguments error
+#pragma PointlessParentCall error
```

Comment:
sadly these aren't actually pointless. (thanks byond)

```dm
/datum/dohelloworld()
	world << "World"
/datum/proc/dohelloworld()
	world << "Hello\..."
	..()
```

will print `Hello World` (warning, last checked in like 509)

Reply:
@MrStonedOne Ask and you shall receive: https://github.com/OpenDreamProject/OpenDream/pull/1713